source/BaselineOfSagan/BaselineOfSagan.class.st

```diff
@@ -78,7 +78,7 @@ BaselineOfSagan >> setUpPackages: spec [
 		group: 'Tests' with: 'Sagan-Core-Tests'.
 
 	spec
-		package: 'Sagan-RDBMS' with: [ spec requires: #('Glorp-Core' 'Core' 'ObjectPool-Core' 'Launchpad-Logging') ];
+		package: 'Sagan-RDBMS' with: [ spec requires: #('Glorp-Core' 'Core' 'ObjectPool-Core' 'Bell-SUnit') ];
 		group: 'RDBMS' with: #('Sagan-RDBMS').
```

Comment:
This package shouldn't be depending on SUnit but on Deployment.

Reply:
You're right, thanks!